src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.4.62
```

Comment:
We have an agreement before that the release/history change should be a separated PR.

Reply:
removed